lib/tasks/cms_categories.rake

```diff
@@ -12,4 +12,11 @@ namespace :cms_categories do
       end
     end
   end
+
+  task destroy: :environment do
+    Comfy::Cms::LayoutsCategory.destroy_all
+    Comfy::Cms::PagesCategory.detroy_all
+    Comfy::Cms::PageCategory.detroy_all
```

Comment:
There's a couple of typos here - also was wondering if we could set this Rake task to run prior to seeds import perhaps in the Comfy monkey patch? I end up forgetting that this task exists which is probably what I should have done before trying to migrate as it's purely dependent on the contents of the YML file

Reply:
Thanks! Fixed the typos.

For clarity, I'm not calling the destroy task in any migration, but just the import one which hopefully should fix the issues you pointend out